src/rsz/src/Resizer.cc

```diff
@@ -1165,7 +1168,15 @@ Resizer::dbuToMeters(int dist) const
 int
 Resizer::metersToDbu(double dist) const
 {
-  return dist * dbu_ * 1e+6;
+  if (dist < 0) {
+    logger_->error(
+        RSZ, 86, "metersToDbu({}) cannot convert negative distances", dist);
+  }
+  // sta::INF is passed to this function in some cases. Protect against
+  // overflow conditions.
```

Comment:
sta::INF is a 1E+30F so this check won't help.  In general I would put the comment before the check rather than after.

Reply:
The comment is unrelated to the check. The check at the top is to prevent people from putting in negative distances.

Whereas the `& INT_MAX` maxes sure that any values are correctly saturated to int max. Which is what the comment is referring to.